demos/funnyThingsApp/gui/gui_conf.ini

```diff
@@ -0,0 +1,17 @@
+[setup] 
+title "Funny Things App" 
+
+[top options] 
+ImageName "images/icub-funny-things.jpg"
+ 
+[right options]
+fileInput "Google key json file (required):" "Choose a file" FILE_INPUT None on None
+dropdownList "Language of the speech:" "" LANGUAGE_SYNTHESIS_INPUT en-US/it-IT/pt-PT/fr-FR/en-GB on None
+dropdownList "Voice types" "" VOICE_NAME_INPUT en-US-Wavenet-A/en-US-Wavenet-B/en-US-Wavenet-C/en-US-Wavenet-D/en-US-Wavenet-E/en-US-Wavenet-F/en-US-Wavenet-G/en-US-Wavenet-H/en-US-Wavenet-I/en-US-Wavenet-J on None
+audioInput "" "Try your voice!" AUDIO_INPUT /home/laura/teamcode/appsAway/demos/synthesis/audioicon.png on None
```

Comment:
We have to fix the path here

Reply:
@lauracavaliere  `AUDIO_INPUT`. cannot be the directory of your pc...

Reply:
@ilaria-carlini I would suggest that we fix this in this PR, merge then deal with all the other `gui_conf.ini` changes separately in a branch.